pkgs/os-specific/linux/tuxedo-drivers/default.nix

```diff
@@ -7,20 +7,34 @@
   kmod,
   pahole,
   gitUpdater,
+  fetchpatch,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "tuxedo-drivers-${kernel.version}";
-  version = "4.14.0";
+  version = "4.15.4";
 
   src = fetchFromGitLab {
     group = "tuxedocomputers";
     owner = "development/packages";
     repo = "tuxedo-drivers";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-79YZaK8WrHOxSUJWxi4lc+foh4xz3EgRnjw+OrL8yqU=";
+    hash = "sha256-WJeju+czbCw03ALW7yzGAFENCEAvDdKqHvedchd7NVY=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "no-cp-etc-usr.patch";
+      url = "https://github.com/user-attachments/files/22226157/no-cp-etc-usr.patch";
```

Comment:
While this does work at the moment, I am not quite sure if references to github attachments are guaranteed to be valid in the future. For example github might decide to garbage collect them at some point in the future?
probably better to add the patch to nixpkgs (usually in the same directory as the default.nix file) and reference that file here. Something like `patches = [ ./no-cp-etc-usr.patch ]` should work.